### ISSUE TITLE
Rename the project

### ISSUE THREAD

**Issue by markGilchrist** [markGilchrist/RxTestExt#6]
I have scored a colossal home goal here as there is already a pod, released by the community by the same name as this.

I propose we rename this project to something more meaningful. I suggest  as these extensions are on the testObserver we rename this to `TestObserverExt`?

if you cant think of a better name then please use the above

**Comment by andecoder** [comment]
I agree we should rename this project, that was bad from us to not check for name clashes before...
We must not forget to update all references to the old name in the project.

For name suggestion, I thought on `RxTestPlus`

**Comment by markGilchrist** [comment]
@andecoder I have to confess I did check, just not hard enough, which is more embarrassing than not checking!

@sindhu238  and @nckmrnv  do you have any new suggestions and or feeling on `RxTestPlus` or `TestObserverExt`

**Comment by sindhu238** [comment]
How about RxSwiftTestExt?  I kind of liked RxTestPlus.

**Comment by markGilchrist** [comment]
I think most people are going to `RxTestPlus`
